project/msm8974.mk

```diff
@@ -17,6 +17,7 @@ ENABLE_SDHCI_SUPPORT := 1
 ENABLE_USB30_SUPPORT := 1
 
 #DEFINES += WITH_DEBUG_DCC=1
+DEFINES += WITH_DEBUG_LOG_BUF=1
```

Comment:
This should be in `msm8974-secondary.mk` (I try to avoid modifying the original make files).

Reply:
B-but.. this is same as in msm8916.mk currently https://github.com/msm8916-mainline/lk2nd/blob/master/project/msm8916.mk#L34

Reply:
Yeah but for some reason Qualcomm added it there :D

Reply:
:ok: